community/repmgr/APKBUILD

```diff
@@ -31,15 +31,12 @@ check() {
 package() {
 	cd "$builddir"
 
-	make USE_PGXS=1 DESTDIR="$pkgdir" install
+	./configure && make USE_PGXS=1 DESTDIR="$pkgdir" install
```

Comment:
do you really need to re-run configure before installing it? that does not seems right

Reply:
I din't (don't) fully understand the build process. But I strongly guess it's not necesary.. :+1: